components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpointTest.java

```diff
@@ -158,6 +158,7 @@ protected Map<String, Object> initialValue() {
         if ("foo.com".equals(tenantDomain)) {
             when(OAuth2Util.mapSignatureAlgorithmForJWSAlgorithm("SHA512withRSA")).thenReturn(JWSAlgorithm.RS256);
         }
+        when(OAuth2Util.getThumbPrint(any(), anyString())).thenReturn("dummyThumbPrintValue");
```

Comment:
shall we add unit test for x5c and x5t as well? https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1880/files#diff-78b8adb9516abd326faa725cb5273b3138e8d2eb6fda078d7dac4026751c38abR175